src/MessageWrapper.php

```diff
@@ -0,0 +1,94 @@
+<?php
```

Comment:
SHOULD:  PSR-12 requires an empty line after the opening tag and before the namespace declaration.  We probably want to adopt that as the code standard since PSR-2 is depreciated now.  (This applies here and throughout the PR)